pkg/cmd/server/kubernetes/node/options/options.go

```diff
@@ -84,6 +84,13 @@ func ComputeKubeletFlags(startingArgs map[string][]string, options configapi.Nod
 	setIfUnset(args, "authorization-webhook-cache-authorized-ttl", options.AuthConfig.AuthorizationCacheTTL)
 	setIfUnset(args, "authorization-webhook-cache-unauthorized-ttl", options.AuthConfig.AuthorizationCacheTTL)
 
+	// Override kubelet iptables-masquerade-bit value to match overridden kube-proxy
+	// iptables-masquerade-bit value, UNLESS the user has overridden kube-proxy to match the
+	// previously-not-overridden kubelet value, in which case we don't want to re-break them.
+	if len(options.ProxyArguments["iptables-masquerade-bit"]) != 1 || options.ProxyArguments["iptables-masquerade-bit"][0] != "14" {
```

Comment:
Don't we want to override this flag only when they are using openshift SDN?
[i.e. when network.IsOpenShiftNetworkPlugin(options.NetworkConfig.NetworkPluginName) == true]

Reply:
@pravisankar no, looks like the defaulting to 0 is done unconditionally for kube-proxy config in pkg/cmd/server/kubernetes/network/options/options.go which is called from start_node.go's StartNode() whether openshift-sdn is enabled or not.